src/test/unit/components/GameCard.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { GameCard } from '../../../components/GameCard';
+
+// const img = "path-to-image"
```

Comment:
do you need this comment?

Reply:
Not strictly but I figured I'd leave it in as a placeholder for when we actually implement it